src/docs/releases/1.0.0-rc1.md

```diff
@@ -0,0 +1,242 @@
+# Orchard Core 1.0.0-rc1
```

Comment:
As I've suggested under https://github.com/OrchardCMS/OrchardCore/issues/5841 the O1 release notes format I think is pretty good. While it's not entirely applicable a pretty much mechanical release notes like this is not sufficient I think: The long list of enhancements might be a good reference but if we want people to get a general understanding of what's new it won't work.

While I don't see a point on going back to betas and doing this but eventually we need to start and probably do that with RC1? The structure I can imagine working is what I've added here: https://github.com/OrchardCMS/OrchardCore/blob/62c218ae7ed7d3354db66e588e5ee2652c9ae2be/src/docs/topics/publishing-releases/README.md "Create release notes..."

Reply:
Yes, we can add more details to the release notes than just the generated PRs links:
- Breaking Changes / Migration
- Enhancements / New features
- Pre-requesites (Framework version)

Reply:
So like this?

>   - Overview of the release's highlights and goals. What do you want people to remember this release for?
>  - Prerequisites. What framework version do you need, anything else to work with Orchard?
>  - Upgrade steps, any migration necessary from previous versions.
>  - Full changelog. You can generate this with [github-changelog](https://github.com/cfpb/github-changelog) with the `changelog OrchardCMS OrchardCore <previous version> <current version>` command, e.g. `changelog OrchardCMS OrchardCore 1.0.0-rc1 1.0.0-rc2`.

Reply:
Exactly.